### DISCUSSION TITLE
{{data 'bucketNameOrId' 'path.to.property'}} doesnt work

### DISCUSSION THREAD

**Discussion by alindbach** [mockoon/mockoon#1586] [Help] [answered]
I have a databucket called fakerValues:

```
{
  "fakerValuesList": [
    {
      "indBenefitAmount": {{faker 'number.float' fractionDigits=2 min=0 max=500}}
    },
    {
      "indBenefitAmount": {{faker 'number.float' fractionDigits=2 min=0 max=500}}
    },
    {
      "indBenefitAmount": {{faker 'number.float' fractionDigits=2 min=0 max=500}}
    },
    {
      "indBenefitAmount": {{faker 'number.float' fractionDigits=2 min=0 max=500}}
    }
  ]
}

```

And im trying to get the first index of this list by doing as is stated in the documentation:

{{data 'bucketNameOrId'}}

{{data 'bucketNameOrId' 'path.to.property'}}
{{data 'bucketNameOrId' 'deep.property\.with\.dot'}}

{{data 'bucketNameOrId' '$.array.[*].property'}}

However this outputs the whole list:
"firstBenefit": {{data 'fakerValues' '$.fakerValuesList.0.indBenefitAmount'}}

How do i get the first index of the data bucket?

**Comment by alindbach** [comment, marked as answer]
I had a fault in my code, my data bucket had a trailling comma. Closing the discussion.

**Reply by 255kb** [reply]
Hi, glad you found the issue. It's something I plan on improving for the next release: https://github.com/mockoon/mockoon/issues/1581
